packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -561,6 +561,63 @@ export class PipelineEditor extends React.Component<
     });
   }
 
+  async handleLoadPipeline(): Promise<void> {
+    this.widgetContext.ready.then(() => {
+      let pipelineJson = this.widgetContext.model.toJSON();
+      const pipelineVersion: number =
+        +Utils.getPipelineAppdataField(pipelineJson, 'version') || 0;
+      if (pipelineVersion !== PIPELINE_CURRENT_VERSION) {
+        // pipeline version and current version are divergent
+        if (pipelineVersion > PIPELINE_CURRENT_VERSION) {
+          // in this case, pipeline was last edited in a "more recent release" and
+          // the user should update his version of Elyra to consume the pipeline
+          showDialog({
+            title: 'Load pipeline failed !',
+            body: (
+              <p>
+                The current pipeline was created/edited with a more recent
+                version of Elyra.
+                <br />
+                Upgrade Elyra before editing this pipeline.
+              </p>
+            ),
+            buttons: [Dialog.okButton()]
+          });
+          return;
+        } else {
+          // in this case, pipeline was last edited in a "old" version of Elyra and
+          // it needs to be updated/migrated.
+          showDialog({
+            title: 'Migrate pipeline ?',
```

Comment:
Space before question mark.
```suggestion
            title: 'Migrate pipeline?',
```